tests/test_mssqlcliclient.py

```diff
@@ -202,6 +203,37 @@ def test_stored_proc_multiple_result_sets(self):
         finally:
             shutdown(client)
 
+    
+    def test_select_result_unicode_encoded(self):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        db_name = 'db_' + random_str()
```

Comment:
>db_name [](start = 8, length = 7)

While we're at it, should we test a unicode database name?

Reply:
Creating a test database is removed from the unit test code. This line no longer exists.

Reply:
This is still something we need to test. Let's add that in another PR.